aws/ec2.go

```diff
@@ -729,6 +743,16 @@ func (sg DefaultSecurityGroup) nuke() error {
 				return fmt.Errorf("error deleting eggress rule: %s", errors.WithStackTrace(err))
 			}
 		}
+
+		ipv6EgressRule := sg.getDefaultSecurityGroupIPv6EgressRule()
+		_, err = sg.svc.RevokeSecurityGroupEgress(ipv6EgressRule)
+		if err != nil {
+			if awsErr, isAwsErr := err.(awserr.Error); isAwsErr && awsErr.Code() == "InvalidPermission.NotFound" {
```

Comment:
Error handling seems to be similar to the handling of `ingressRule / egressRule` errors , was wondering if will be helpful if handling can be extracted to a common function?

Reply:
Yes, good point. I will leave as it is since it's outside the scope of this PR, and open an issue! Thanks for the review